### PR TITLE
Master

### PR DESCRIPTION
**Pull request by julieplummer20**

### CODE REVIEW

File: tutorials/abap-environment-external-api/abap-environment-external-api.md [hunk 1]
```diff
@@ -1,32 +1,180 @@
...
+## You will learn
 
+- How to connect to an external RESTful API, in this case the Overpass street map service: [http://overpass-api.de/](http://overpass-api.de/), by creating the necessary communication artifacts
+- How to work with HTTP requests in your ABAP code
+- How to query the service
+- How to parse the information in an ABAP class using either JSON and SAP's XCO Library or using XML and a simple transformation
+- How to create an executable class and display the output in the ABAP console
+This tutorial enables you to work with data from an external RESTful API in your ABAP applications.
+
+This tutorial was written for SAP BTP ABAP Environment. However, you should also be able to use it in SAP S/4HANA Cloud Environment in the same way.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api/abap-environment-external-api.md:32]: I would format this as a note.

File: tutorials/abap-environment-external-api/abap-environment-external-api.md [hunk 2]
```diff
@@ -81,6 +229,14 @@ Now, create the **Simple Transformation**, which converts the XML data to ABAP d
 
 6. Format (`Ctrl+F1`), save (`Ctrl+S`) and activate (`Ctrl+F3`) your class.    
 
+[OPTION END]
+
+
+[OPTION BEGIN [JSON]]
+If you are using JSON, skip this step.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api/abap-environment-external-api.md:236]: I would format this as a note.

File: tutorials/abap-environment-external-api/abap-environment-external-api.md [hunk 3]
```diff
@@ -104,33 +260,25 @@ The skeleton class is displayed in a new editor.
...
-      INTERFACES if_oo_adt_classrun.
-    PRIVATE SECTION.
-
-    ```
+### Copy code
 
-This enables you to run the class in the console.
+[OPTION BEGIN [XML]]
 
+To be able to access the external service, you must:
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api/abap-environment-external-api.md:267]: If the order of these steps is relevant, please use an ordered list.

File: tutorials/abap-environment-external-api/abap-environment-external-api.md [hunk 4]
```diff
@@ -218,28 +366,143 @@ To be able to access the external service, you must:
 
     ```
 
-2. Check your syntax (`Ctrl+F2`).
+[OPTION END]
+
+
+[OPTION BEGIN [JSON]]
+
+To be able to access the external service, you must:
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-external-api/abap-environment-external-api.md:374]: If the order of these steps is relevant, please use an ordered list.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by jmmargo** [comment]
Julie, in step 10, there should be a line break between the first line of text and the first image.  They are currently appearing side-by-side.  You can add an empty <div></div> tag in between if a line break itself does not work.

**Comment by julieplummer20** [comment]
Done

**Review by julieplummer20 — COMMENTED** [review]
Fixed link; fixed ordered lists

**Pull request merged**